bundles/org.openhab.binding.hue/README.md

```diff
@@ -84,7 +101,7 @@ discovery.hue:background=false
 Once it is added as a Thing, its authentication button (in the middle) needs to be pressed in order to authorize the binding to access it.
 Once the binding is authorized, it automatically reads all devices and groups that are set up on the Hue Bridge and puts them into the Inbox.
 
-## Thing Configuration
+## Bridge Thing Configuration (CLIP v2)
```

Comment:
Consider merging this chapter with the next and just highlight differences and/or create sub chapters when they differ too much.

Reply:
> Consider merging this chapter with the next 
> create sub chapters when they differ too much.

I did consider. And (THEREFORE) I did create two chapters, because they do indeed differ too much.